app/views/profiles/show.html.erb

```diff
@@ -148,6 +148,7 @@
         <% end %>
       </div>
     <% end %>
+    </div>
```

Comment:
Surprising!

Reply:
Oh that's actually a vestigial holdover - shouldn't be there!

Reply:
[309f02a](https://github.com/broadinstitute/single_cell_portal_core/pull/1979/commits/309f02ac5945957b77172d2b36feef535c4b063b)